Modules/Finance/Finance.cs

```diff
@@ -12,6 +12,8 @@ public static class FinanceModule
     {
         private const string module = "finance";
 
+		public static readonly ExportSubModule Accounting = new ExportSubModule(module, "accounting");
```

Comment:
Just going to nitpick what looks like some extra whitespace here.

Reply:
Dude, you want contribution on this? I don't see extra whitespace :)